src/main/java/core/basesyntax/suppliers/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.model.Ball;
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(supplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
the same for random. Also, let's avoid using magic numbers, as described in the common mistakes